platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/handle_mgmt_interface.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+#Due to the hardware design, as4630-54pe use "eth2" instead of "eth0" as management interface.
+#Rename netdev "eth0" and "eth2" to swap original "eth2" to "eth0".
+
+ifconfig eth0 down
```

Comment:
can you use udev rule to rename the eth0?

Reply:
Hi lguohan:
Yes!! I have tried to use udev rule, and work good.
But, I found out  the all of the project in Accton all use same architecture. 
If I use /etc/udev/rules.d/70-persistent-net.rules to change eth2 to eth0, 
our command code (TOPDIR/platform/broadcom/sonic-platform-modules-accton/debian/rules) need to add a rule to copy udev/rules.d/70-persistent-net.rules.
As our internal-discussion, we do not want to modify common code(ex: rules).
So, can we help me merge this issue into the project. This issue is emergency.

Common code modify Example:
(for mod in $(MODULE_DIRS); do \
dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} $(KERNEL_SRC)/$(INSTALL_MOD_DIR); \
dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} usr/local/bin; \
dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} lib/systemd/system; \
**dh_installdirs -p$(PACKAGE_PRE_NAME)-$${mod} etc/udev/rules.d;**
cp $(MOD_SRC_DIR)/$${mod}/$(MODULE_DIR)/*.ko debian/$(PACKAGE_PRE_NAME)-$${mod}/$(KERNEL_SRC)/$(INSTALL_MOD_DIR); \
cp $(MOD_SRC_DIR)/$${mod}/$(UTILS_DIR)/* debian/$(PACKAGE_PRE_NAME)-$${mod}/usr/local/bin/; \
cp $(MOD_SRC_DIR)/$${mod}/$(SERVICE_DIR)/*.service debian/$(PACKAGE_PRE_NAME)-$${mod}/lib/systemd/system/; \
**cp $(MOD_SRC_DIR)/$${mod}/udev/* debian/$(PACKAGE_PRE_NAME)-$${mod}/etc/udev/rules.d/;**

Reply:
can you create an issue in buildimage and assign to accton, to make you folks will come back to use udev rule?

Reply:
> 
> 
> can you create an issue in buildimage and assign to accton, to make you folks will come back to use udev rule?

In our internal discussion ,  supervisor agree the modification in TOPDIR/platform/broadcom/sonic-platform-modules-accton/debian/rules . I will use udev style to solve this issue. Thank you for your suggestion.

Reply:
retest this please

Reply:
retest
 please

Reply:
reset please